demo/python/pv_recorder_demo.py

```diff
@@ -39,7 +39,7 @@ def main():
     args = parser.parse_args()
 
     if args.show_audio_devices:
-        devices = PvRecorder.get_audio_devices()
+        devices = PvRecorder.get_available_devices()
```

Comment:
notes for code below:
- "pvrecorder.py version..." -> "pvrecorder version.."
- pcm -> frame
- use sample rate from class, not hard-coded 16kHz
- frame_length can be a constant and shared between wavfile and pvrecorder